### ISSUE TITLE
[BUG] Problem in routes.razor with new modules

### ISSUE THREAD

**Issue by allschu** [oqtane/oqtane.framework#4562]
### Oqtane Info

Version - 5.2.1
Render Mode - Static 
Interactivity - Server
Database - Postgres

### Describe the bug
Maybe this bug is already familiar but I couldn't find any answers. My apologies. Just point me to the right topic

When you start a new module (nothing fancy, just a new one, as default) and you publish the module to a page I get the following error. 

System.InvalidOperationException: 'Cannot provide a value for property 'newsService' on type 'Schuurmans.Module.news.Index'. There is no registered service of type 'Schuurmans.Module.news.Services.InewsService'.'

It looks like it's trying to load the index page, but it cannot resolve the Service, which is needed as a dependency. Is it not automatically registered? 

<img width="653" alt="image" src="https://github.com/user-attachments/assets/d27d0375-87e3-4d78-a0e6-caf6495de632">

### Expected Behavior
Show the index page of the newly created module

### Steps To Reproduce
Get the latest version

**Comment by leigh-pointer** [comment]
Hi @allschu, did you use the Oqtane templates to scaffold the module?  I have just built a simple module using the Templates and have no issue.

**Comment by allschu** [comment]
@leigh-pointer yes I used the default templates. I found out what the problem is.
In version 5.1.0 I got a Startup class (in the module server project) in which the following registration are present

```
  services.AddTransient<INewsService, ServerNewsService>();
  services.AddDbContextFactory<NewsContext>(opt => { }, ServiceLifetime.Transient);
```
But in version 5.2.1 when I follow the same action to create a new Module, I don't get this startup class by default anymore.

**Comment by leigh-pointer** [comment]
@allschu correct, the templates were change in #4541

**Comment by allschu** [comment]
@leigh-pointer if I add these registrations in a default clean 5.2.1 installation with my own serverstartup class everything works.

**Comment by sbwalker** [comment]
The IServerStartup class should not have been deleted as part of #4541 - the fix was only intended to address the issue reported in the Client project - I am not sure how the file in the Server project got deleted.

**Comment by sbwalker** [comment]
The ServerStartup.cs file is in my local environment - however for some reason Git removed it as part of #4541 and is not allowing me to add it back

![image](https://github.com/user-attachments/assets/89b8b6d0-6dda-4504-89d1-f438e9ea452e)

Once I figure out my local Git problem I should be able to add the file back to the template

**Comment by sbwalker** [comment]
Resolved in #4565:

PR https://github.com/oqtane/oqtane.framework/pull/4541 was not valid as it did not actually include the necessary changes to resolve issue https://github.com/oqtane/oqtane.framework/issues/4537 - this was because the gitignore file was excluding files which should have been included. This PR fixes the gitignore and merges the default module template files which were supposed to be committed in https://github.com/oqtane/oqtane.framework/pull/4541

**Comment by dvdobrovolskiy** [comment]
sorry. nope, used release version. Please close issue
